github/activity_events.go

```diff
@@ -48,6 +48,8 @@ func (e *Event) ParsePayload() (payload interface{}, err error) {
 		payload = &DeploymentStatusEvent{}
 	case "ForkEvent":
 		payload = &ForkEvent{}
+	case "AppAuthorizationEvent":
```

Comment:
Once #1057 is merged, I would like this to also be moved to `event.go`... but I can do that if this PR finishes first.

Reply:
Also, can we please keep these event names in lexicographical sorted order?

Reply:
Would be good to add a comment to this switch statement if we would like it to be ordered.